supervisord.conf

```diff
@@ -0,0 +1,32 @@
+[supervisord]
+nodaemon=true
+loglevel=warn
+logfile_maxbytes=0
+logfile=/dev/stderr
+
+[program:nginx]
+command = nginx -g 'daemon off;'
+autostart = true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[program:php-fpm7.3]
+command = /usr/sbin/php-fpm7.3 -F
+autostart = true
+stdout_logfile=/dev/stdout
+stdout_logfile_maxbytes=0
+stderr_logfile=/dev/stderr
+stderr_logfile_maxbytes=0
+
+[program:foo]
+directory = /var/www/app/wordpress
+command = sh -c 'sleep 7; ./wp core install --url=%(ENV_BLOG_URL)s --title="%(ENV_BLOG_TITLE)s" --admin_user=%(ENV_ADMIN_USER)s --admin_email=%(ENV_ADMIN_EMAIL)s --admin_password=%(ENV_ADMIN_PASSWORD)s --allow-root; ./wp theme install %(ENV_WP_THEME)s --activate --allow-root; ./wp plugin install %(ENV_PLUGIN_URL)s --activate --allow-root'
+autostart = true
```

Comment:
why do you need this line?

Reply:
Well, ```autostart=true``` will allow the program ```foo``` to be autoamtically executed when suprevisord starts. If set as ```false``` I'd have to start the program manually (i.e., with supervisorctl)

Reply:
ok